molo/surveys/migrations/0021_surveyresponserule.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2017-11-22 15:09
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+import modelcluster.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('surveys', '0020_formfield_admin_label'),
+        ('wagtail_personalisation', '0016_auto_20171122_1924')
```

Comment:
@tm-kn This migration doesn't exist in the package. The last migration is  `0015_static_users.py`. I think it may have been generated by running `makemigrations`. I'm not sure why it would do that though.

Reply:
The migration on the wagtail-personalisation package may provide some insight? ¯\_(ツ)_/¯

Reply:
That's indeed true. `makemigrations` generated that migration...
```
Migrations for 'wagtail_personalisation':
  0016_auto_20171123_1702.py:
    - Alter field segment on dayrule
    - Alter field segment on devicerule
    - Alter field segment on queryrule
    - Alter field segment on referralrule
    - Alter field segment on timerule
    - Alter field segment on userisloggedinrule
    - Alter field segment on visitcountrule
```